Mathlib/Probability/Kernel/Invariance.lean

```diff
@@ -76,12 +76,12 @@ def Invariant (κ : kernel α α) (μ : Measure α) : Prop :=
 
 variable {κ η : kernel α α} {μ : Measure α}
 
-theorem Invariant.def (hκ : Invariant κ μ) : μ.bind κ = μ :=
+theorem Invariant.def' (hκ : Invariant κ μ) : μ.bind κ = μ :=
```

Comment:
```suggestion
-- Adaptation note: 2024-03-15
-- Renamed to avoid the reserved name `Invariant.def`.
theorem Invariant.def' (hκ : Invariant κ μ) : μ.bind κ = μ :=
```